bokehjs/src/lib/models/widgets/date_picker.ts

```diff
@@ -61,36 +27,14 @@ export class DatePickerView extends InputWidgetView {
     this.input_el = input({type: "text", class: bk_input, disabled: this.model.disabled})
     this.group_el.appendChild(this.input_el)
 
-    this._picker = new Pikaday({
-      field: this.input_el,
-      defaultDate: this._unlocal_date(new Date(this.model.value)),
-      setDefaultDate: true,
-      minDate: this.model.min_date != null ? this._unlocal_date(new Date(this.model.min_date)) : undefined,
-      maxDate: this.model.max_date != null ? this._unlocal_date(new Date(this.model.max_date)) : undefined,
-      onSelect: (date) => this._on_select(date),
-    })
-
-    this._root_element.appendChild(this._picker.el)
-  }
-
-  _unlocal_date(date: Date): Date {
-    //Get the UTC offset (in minutes) of date (will be based on the timezone of the user's system).
-    //Then multiply to get the offset in ms.
-    //This way it can be used to recreate the user specified date, agnostic to their local systems's timezone.
-    const timeOffsetInMS = date.getTimezoneOffset() * 60000
-    date.setTime(date.getTime() - timeOffsetInMS)
-
-    const datestr = date.toISOString().substr(0, 10)
-    const tup = datestr.split('-')
-    return new Date(Number(tup[0]), Number(tup[1])-1, Number(tup[2]))
+    this._picker = flatpickr(this.input_el, {
+      onChange: (selectedDates, dateStr, instance) => this._on_change(selectedDates, dateStr, instance),
```

Comment:
Argument names aren't meaningful in JS, as they are e.g. in Python, so they can be safely renamed to snake-case to match bokeh's naming convention.

Reply:
I  know :) This is very early wip